server/mqtt.go

```diff
@@ -159,6 +164,9 @@ const (
 
 	// This is how frequently the timer to cleanup the sessions flappers map is firing.
 	mqttSessFlappingCleanupInterval = 5 * time.Second
+
+	// Default retry delay if transfer of old session streams to new one fails
+	mqttDefaultTransferRetry = 5 * time.Second
```

Comment:
Might take longer.. If you try multiple times no worries.

Reply:
This is the interval between retries (but does not count the time of a transfer itself). Not sure what you are asking me to change here? Make the interval longer?

Reply:
Might make sense to do 10s, but where it is now is good too.